js/viz/chart.d.ts

```diff
@@ -227,6 +227,13 @@ export interface baseLabelObject {
      * @public
      */
     getBoundingRect(): any;
+    /**
+     * @docid
+     * @publicName shift(x, y)
+     * @return object
+     * @public
+     */
+    shift(x: number, y: number): any;
```

Comment:
method returns _baseLabelObject_ . see:  https://github.com/DevExpress/DevExtreme/blob/23_1/js/viz/series/points/label.js#L514
May be we concretize this type here?

Reply:
Yes, I agree with it. Fixed here: [755efe07d4738719c9f1e1727edd9e166bdad066].